### ISSUE TITLE
Make fails on Linux

### ISSUE THREAD

**Issue by chrisbeckstrom** [VCVRack/Rack#7]
Building dependencies works fine (no errors), but when I run `make` I get this:

```
Rack$ make
g++ -Iinclude -Idep/include -Idep/lib/libzip/include -DVERSION=dev -MMD -O3 -march=core2 -ffast-math -g -Wall -DARCH_LIN -std=c++11 -c -o build/src/dsp.cpp.o src/dsp.cpp
In file included from include/dsp.hpp:7:0,
                 from src/dsp.cpp:1:
include/math.hpp: In member function ‘bool rack::Vec::isFinite()’:
include/math.hpp:183:20: error: ‘isfinite’ was not declared in this scope
   return isfinite(x) && isfinite(y);
                    ^
include/math.hpp:183:20: note: suggested alternative:
In file included from /usr/include/c++/5/complex:44:0,
                 from include/dsp.hpp:6,
                 from src/dsp.cpp:1:
/usr/include/c++/5/cmath:601:5: note:   ‘std::isfinite’
     isfinite(_Tp __x)
     ^
compile.mk:46: recipe for target 'build/src/dsp.cpp.o' failed
make: *** [build/src/dsp.cpp.o] Error 1
```

System info:
```
$ lsb_release -a
LSB Version:	core-9.20160110ubuntu0.2-amd64:core-9.20160110ubuntu0.2-noarch:security-9.20160110ubuntu0.2-amd64:security-9.20160110ubuntu0.2-noarch
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.2 LTS
Release:	16.04
Codename:	xenial
```

**Comment by AndrewBelt** [comment]
Thanks, I'll take a look at this in Ubuntu.

**Comment by tomchiverton** [comment]
This is critical to your wider adoption on Linux. The binary download does not run on the most popular platforms, those based on Ubuntu 17.04 or 16.04.
See #43

**Comment by Malifick** [comment]
Bump

**Comment by AndrewBelt** [comment]
Fixed in v0.3.2. Since then, I use Ubuntu 16.04 for production builds.
